From 9dabc7fa742acc1dca15bda9cf4242a827175880 Mon Sep 17 00:00:00 2001
From: Doug Ledford <dledford@redhat.com>
Date: Wed, 24 Sep 2014 14:58:26 -0400
Subject: [infiniband] usnic: Remove unused includes of <linux/version.h>

Message-id: <426af1eed24ae369fcad98da058567cc4b419b8d.1411569661.git.dledford@redhat.com>
Patchwork-id: 96584
O-Subject: [Patch RHEL7 24/29] IB/usnic: Remove unused includes of <linux/version.h>
Bugzilla: 916382
RH-Acked-by: Jes Sorensen <Jes.Sorensen@redhat.com>

From: Wei Yongjun <yongjun_wei@trendmicro.com.cn>

Remove including <linux/version.h> that don't need it.

Signed-off-by: Wei Yongjun <yongjun_wei@trendmicro.com.cn>
Signed-off-by: Roland Dreier <roland@purestorage.com>
(cherry picked from commit a384b20e417ae0f5f1f359600b4bdcc34265b256)
Signed-off-by: Doug Ledford <dledford@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/infiniband/hw/usnic/usnic_ib.h b/drivers/infiniband/hw/usnic/usnic_ib.h
index 111a86e..e5a9297 100644
--- a/drivers/infiniband/hw/usnic/usnic_ib.h
+++ b/drivers/infiniband/hw/usnic/usnic_ib.h
@@ -21,7 +21,6 @@
 
 #include <linux/iommu.h>
 #include <linux/netdevice.h>
-#include <linux/version.h>
 
 #include <rdma/ib_verbs.h>
 
diff --git a/drivers/infiniband/hw/usnic/usnic_uiom_interval_tree.c b/drivers/infiniband/hw/usnic/usnic_uiom_interval_tree.c
index 7e1dafc..d135ad9 100644
--- a/drivers/infiniband/hw/usnic/usnic_uiom_interval_tree.c
+++ b/drivers/infiniband/hw/usnic/usnic_uiom_interval_tree.c
@@ -2,7 +2,6 @@
 #include <linux/list.h>
 #include <linux/slab.h>
 #include <linux/list_sort.h>
-#include <linux/version.h>
 
 #include <linux/interval_tree_generic.h>
 #include "usnic_uiom_interval_tree.h"
diff --git a/drivers/infiniband/hw/usnic/usnic_uiom_interval_tree.h b/drivers/infiniband/hw/usnic/usnic_uiom_interval_tree.h
index 030ba6e..d4f752e 100644
--- a/drivers/infiniband/hw/usnic/usnic_uiom_interval_tree.h
+++ b/drivers/infiniband/hw/usnic/usnic_uiom_interval_tree.h
@@ -19,7 +19,6 @@
 #ifndef USNIC_UIOM_INTERVAL_TREE_H_
 #define USNIC_UIOM_INTERVAL_TREE_H_
 
-#include <linux/version.h>
 #include <linux/rbtree.h>
 
 struct usnic_uiom_interval_node {
-- 
1.7.1